src/components/ConcurrencyLimitsTable.vue

```diff
@@ -49,5 +51,6 @@
   ]
 
   const concurrencyLimitSubscription = useSubscription(api.concurrencyLimits.getConcurrencyLimits)
-  const concurrencyLimits = computed(() => concurrencyLimitSubscription.response ?? [])
+  const concurrencyLimits = computed(() => concurrencyLimitSubscription.response)
+  const empty = computed(()=> concurrencyLimits.value && !concurrencyLimits.value.length)
```

Comment:
Usually we check for `loading` and `empty. See [WorkspaceFlows](https://github.com/PrefectHQ/nebula-ui/blob/main/src/pages/WorkspaceFlows.vue#L34)